RealmSwift/MongoClient.swift

```diff
@@ -263,21 +277,21 @@ extension MongoCollection {
     ///   - completion: The resulting bson or error if one occurs
     public func findOneDocument(filter: Document,
                                 _ completion: @escaping MongoFindOneBlock) {
-        guard let rlmBSON = ObjectiveCSupport.convert(object: AnyBSON(filter)) as? [String : RLMBSON] else {
+        guard let rlmBSON = ObjectiveCSupport.convert(object: AnyBSON(filter)) as? [String: RLMBSON] else {
```

Comment:
These guards should really never fail– any reason we are throwing this error?

Reply:
Error is no longer needed with new bson conversion method